### PR TITLE
update installation warning banner & installation instructions

### PR DESCRIPTION
**Pull request by jeromedockes**

### CODE REVIEW

File: doc/announcement.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="sidebar-message">
-    <strong>
-        This package is a work in progress. It hasn't been released yet.
-        To use it anyway, <a href="https://skrub-data.org/stable/install.html">install it from source</a>.
-    </strong>
+    <strong>This package is a 🚧 work in progress ⚠️.</strong>
```
Comment (jeromedockes) [inline doc/announcement.html:2]: for some reason that part gets loaded directly from the repo with javascript when the page is rendered; so we will not see it in the rendered docs until the PR is merged in main. but you can check the wording here

Comment (jeromedockes) [reply doc/announcement.html:2]: @GaelVaroquaux  if you want to check the message

**Comment by GaelVaroquaux** [comment]
> @GaelVaroquaux if you want to check the message

LGTM

File: doc/announcement.html (hunk 1 above)
Comment (Vincent-Maladiere) [reply doc/announcement.html:2]: I'm not sure about this one; all packages are work in progress. The explainer right after is more explicit.

File: doc/install.rst [hunk 2]
```diff
@@ -17,54 +17,44 @@ Installing
...
-        <div class="tab-pane fade" id="conda-tab-pane" role="tabpanel" aria-labelledby="conda-tab" tabindex="0">
+    </div>
+          <div class="tab-pane fade" id="conda-tab-pane" role="tabpanel" aria-labelledby="conda-tab" tabindex="0">
             <hr />
 
-.. warning::
+.. code:: console
 
-   skrub has not yet been released. See the "From source" tab.
+    $ conda install skrub
```
[the remark this answers is not in the document]
Comment (jeromedockes) [reply doc/install.rst:40]: btw does this mean we need to package skrub for conda-forge? https://conda-forge.org/docs/maintainer/adding_pkgs.html#contributing-packages 
or maybe it should just be installable with pip for now?

**Pull request merged**